fix: do not modify property names

zi opened the pull request:
Previuosly, when a property name contained the character "-" it converted the name to camel case.
Now all the property name are strings and remain unchanged

toomuchdesign commented:
Hi @zi,
do we any test covering the specific change introduced by this PR or since we're just bringing back a default behaviour there's no need of it?

PS.
Unfortunately I was quite disoriented by `fp-ts` methods, I have no clue what they do and the docs were not reassuring :)

zi commented:
> do we any test covering the specific change introduced by this PR or since we're just bringing back a default behaviour there's no need of it?

hi @toomuchdesign, yes, that's exactly the case. It's the default behaviour so it's covered by the current tests, i've updated them. 
I removed a test that covered a special case in which we were using this behaviour

toomuchdesign reviewed (approved):
I don't have the full picture, but the bits I could understand looked good to me 👍
